dependencies.yaml

```diff
@@ -123,6 +123,7 @@ main_requires:
   perl(ExtUtils::testlib):
   perl(Fcntl):
   perl(File::Basename):
+  perl(File::chdir):
```

Comment:
The specfile should be updated as well. You can run the `update-deps` target for that.

Reply:
Should be fixed now